counter/schemas.py

```diff
@@ -26,7 +29,72 @@ class Meta:
         fields = ["id", "name"]
 
 
-class ProductSchema(ModelSchema):
+class ProductTypeSchema(ModelSchema):
+    class Meta:
+        model = ProductType
+        fields = ["id", "name", "description", "comment", "icon", "order"]
+
+    url: str
+
+    @staticmethod
+    def resolve_url(obj: ProductType) -> str:
+        return reverse("counter:producttype_edit", kwargs={"type_id": obj.id})
```

Comment:
```suggestion
        return reverse("counter:product_type_edit", kwargs={"type_id": obj.id})
```
À changer un peu partout